### PR TITLE
/availability - Add ability to set as default in the dropdown

### LINKED ISSUE #4443: /availability - Add ability to set as default in the dropdown.
CleanShot 2022-09-13 at 18 38 58@2x

### PR DESCRIPTION
**Pull request by gitstart**
## What does this PR do? 

Adds the ability to set as default in the dropdown. 

Fixes #4443 

Loom Video: https://www.loom.com/share/4a1cefe2437541a7ac4bf647edba2e8e

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by gitstart** [comment]
Hi @Jaibles, kindly review the PR

File: packages/features/schedules/components/ScheduleListItem.tsx [hunk 1]
```diff
@@ -67,22 +83,55 @@ export function ScheduleListItem({
...
+                  }}>
+                  <Label
+                    htmlFor="hiddenSwitch"
+                    className={classNames(
+                      "mt-2 mr-2 self-center sm:inline",
+                      schedule.isDefault ? "cursor-not-allowed" : "cursor-pointer"
+                    )}>
+                    {t("set_to_default")}
+                  </Label>
+                  <Switch checked={schedule.isDefault} id="hiddenSwitch" disabled={schedule.isDefault} />
```
Comment (zomars) [inline packages/features/schedules/components/ScheduleListItem.tsx:113]: We should not have switches inside dropdown

**Review by zomars — CHANGES_REQUESTED** [review]

File: packages/features/schedules/components/ScheduleListItem.tsx (hunk 1 above)
Comment (gitstart) [reply packages/features/schedules/components/ScheduleListItem.tsx:113]: Should it be just text like in the screenshot? cc: @Jaibles 
Screenshot 2022-11-29 at 1 11 59 PM
Screenshot 2022-11-29 at 1 16 30 PM

Comment (emrysal) [reply packages/features/schedules/components/ScheduleListItem.tsx:113]: @Jaibles The implementation is as per spec, but UX wise it just doesn't quite work:
https://www.loom.com/share/4d65f3fad42c4fd4b07d16848f05d0f1

Comment (gitstart) [reply packages/features/schedules/components/ScheduleListItem.tsx:113]: Does this need a redesign or is it enough to just remove the switch button?

Comment (zomars) [reply packages/features/schedules/components/ScheduleListItem.tsx:113]: @gitstart let's remove the toggle, the default schedule should not have a "Set as default" button at all. Let's just show the button on the non default ones.

**Comment by gitstart** [comment]
Hi @zomars, PR has been updated with the requested changes, please take a look

**Review by zomars — APPROVED** [review]
Perfect! Thank you for your contribution 🙏 

Ship it

**Pull request merged**